Hunting Queries/Corelight/CorelightDataTransferedByIp.yaml

```diff
@@ -0,0 +1,26 @@
+id: c5f3974b-f411-4a06-a3f8-ee14fcadd0e1
+name: Corelight - Top sources of data transferred
+description: |
+  'Query searches for top sources by transferred data over period of time.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated